Apps/phvisualping/visualping.json

```diff
@@ -0,0 +1,536 @@
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string",
+                    "example_values": [
+                        "Url: https://example.com/, Description: Example description, Lastcheck: 2020-12-23 01:28:27, Percentdifference: 0.03"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.url",
+                    "data_type": "string",
+                    "example_values": [
+                        "https://example.com/"
+                    ],
+                    "contains": [
+                        "url"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.lastCheck",
+                    "data_type": "string",
+                    "example_values": [
+                        "2020-12-23 01:28:27"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.description",
+                    "data_type": "string",
+                    "example_values": [
+                        "Example description"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.percentDifference",
+                    "data_type": "string",
+                    "example_values": [
+                        "0.03"
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "get images",
+            "description": "Get the images from the last run of a specific job",
+            "verbose": "Get the images from the last run of a specific job, including the previous image and the difference.",
+            "type": "investigate",
+            "identifier": "get_images",
+            "read_only": true,
+            "parameters": {
+                "id": {
+                    "description": "ID of the job",
+                    "data_type": "string",
+                    "required": true,
+                    "order": 0
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.id",
+                    "data_type": "string",
+                    "column_name": "ID",
```

Comment:
Here too, I would update the table outputs with more related information

Reply:
Fixed